File: Fib.java
public class Fib {
  public static void main(String[] args) {
    long start = System.currentTimeMillis();
    nth(40);
    long end = System.currentTimeMillis();
    System.out.printf("%d%n", end - start);
  }

  static int nth(int n) {
    return n < 2 ? n : nth(n - 1) + nth(n - 2);
  }
}

File: fib.c
#include <stdio.h>
#include <time.h>

int fib(int n)
{
  return n < 2 ? n : fib(n - 1) + fib(n - 2);
}

int main()
{
  clock_t start = clock(), diff;
  fib(40);
  diff = clock() - start;

  int msec = diff * 1000 / CLOCKS_PER_SEC;

  printf("%d\n", msec);
  return 0;
}
